src/Tools/dotnet-user-jwts/src/Resources.resx

```diff
@@ -246,6 +246,9 @@
   <data name="KeyCommand_ForceOption_Description" xml:space="preserve">
     <value>Don't prompt for confirmation before resetting the signing key.</value>
   </data>
+  <data name="KeyCommand_IssuerOption_Description" xml:space="preserve">
+    <value>The issuer associated with the signing key to be deleted. Defaults to 'dotnet-user-jwts'.</value>
```

Comment:
The `key` command states "Display or reset the signing key used to issue JWTs", these new options should probably say something similar instead of just delete.
"The issuer associated with the signing key to be displayed or reset".